pucauto.py

```diff
@@ -153,43 +154,46 @@ def find_and_send_add_ons():
     """
 
     DRIVER.get("https://pucatrade.com/trades/active")
-    DRIVER.find_element_by_css_selector("div.dataTables_filter input").send_keys('Unshipped')
-    # Wait a bit for the DOM to update after filtering
-    time.sleep(5)
-
-    soup = BeautifulSoup(DRIVER.page_source, "html.parser")
-
-    unshipped = set()
-    for a in soup.find_all("a", class_="trader"):
-        unshipped.add(a.get("href"))
-
-    goto_trades()
-    wait_for_load()
-    load_trade_list()
-    soup = BeautifulSoup(DRIVER.page_source, "html.parser")
+    try:
+        DRIVER.find_element_by_css_selector("div.dataTables_filter input").send_keys('Unshipped')
```

Comment:
Instead of wrapping the entire function body in a try/except, can you just wrap this line. Also instead of printing on an exception, just `return`. I suggest this because there is no output in normal cases where /trades/active has cards but addons don't exist.